src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2075,33 +2161,51 @@
+    examples:
+        - name: Start a troubleshooting operation
+          text: |
+            az network watcher troubleshooting start --resource-group MyResourceGroup \\
+                --resource MyResourceName --resource-type {vnetGateway/vpnConnection} --storage-account MyStorageAccount \\
+                --storage-path https://{storageAccountName}.blob.core.windows.net/{containerName}
 """
 
 helps['network watcher troubleshooting show'] = """
     type: command
     short-summary: Get the results of the last troubleshooting operation.
+    examples:
+        - name: Show the results or status of a troubleshooting operation
+          text: az network watcher troubleshooting show  --resource MyResourceName --resource-group MyResourceGroup --resource-type vnetGateway
 """
 
 helps['network watcher test-ip-flow'] = """
     type: command
     short-summary: Test IP flow to/from a VM given the currently configured network security group rules.
-    long-summary: Requires that Network Watcher is enabled for the region in which the VM is located.
+    long-summary: Requires that Network Watcher is enabled for the region in which the VM is located. For more information visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-check-ip-flow-verify-cli
     parameters:
         - name: --local
-          short-summary: The private IPv4 address for the VM's NIC and the port of the packet in
+          short-summary: >
+            The private IPv4 address for the VM's NIC and the port of the packet in
             X.X.X.X:PORT format. '*' can be used for port when direction is outbound.
         - name: --remote
-          short-summary: The IPv4 address and port for the remote side of the packet
+          short-summary: >
+            The IPv4 address and port for the remote side of the packet
             X.X.X.X:PORT format. '*' can be used for port when direction is inbound.
         - name: --direction
           short-summary: Direction of the packet relative to the VM.
         - name: --protocol
           short-summary: Protocol to test.
+    examples:
+        - name: Run test-ip-flow verify to test logical connectivity from a VM to the specified destination IPv4 address and port.
+          text: |
+            az network watcher test-ip-flow --resource-group MyResourceGroup --direction Outbound  \\
+                --protocol TCP --local 10.0.0.4:* --remote 10.1.0.4:80 --vm MyVm
 """
 
 helps['network watcher test-connectivity'] = """
     type: command
-    short-summary: Test if a direct TCP connection can be established between a Virtual Machine and a given endpoint.
+    short-summary: (PREVIEW) Test if a connection can be established between a Virtual Machine and a given endpoint.
+    long-summary: >
+                To register for this feature and additional examples visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-connectivity-cli
```

Comment:
Compound sentence reads as though the user is also registering for examples. Should read `To register for this feature or see examples, visit`

Reply:
Fixed in my branch.